test/testapps/BasicTestApp/wwwroot/js/index.js

```diff
@@ -0,0 +1,176 @@
+// We'll store the results from the tests here
+
+function createMethodOptions(methodName) {
+  return {
+    type: {
+      assembly: "BasicTestApp",
```

Comment:
Minor nitpick: some quote-style inconsistencies in this file. Should be possible to use just backtick and single-quote.

Reply:
Sure